$OpenBSD: patch-content_common_user_agent_cc,v 1.17 2021/05/27 04:21:01 robert Exp $

Append Linux x86_64 to the user agent because we have to lie ...

Index: content/common/user_agent.cc
--- content/common/user_agent.cc.orig
+++ content/common/user_agent.cc
@@ -225,6 +225,16 @@ std::string BuildOSCpuInfoFromOSVersionAndCpuType(cons
   );
 #endif
 
+#if defined(OS_OPENBSD)
+#if defined(__x86_64__)
+  base::StringAppendF(&os_cpu, "; Linux x86_64");
+#elif defined(__aarch64__)
+  base::StringAppendF(&os_cpu, "; Linux aarch64");
+#else
+  base::StringAppendF(&os_cpu, "; Linux i686");
+#endif
+#endif
+
   return os_cpu;
 }
 
